src/modules/mavlink/mavlink_main.cpp

```diff
@@ -681,6 +682,11 @@ int Mavlink::mavlink_open_uart(int baud, const char *uart_name, bool force_flow_
 	case 1500000: speed = B1500000; break;
 #endif
 
+#ifdef B2000000
```

Comment:
Where is this missing?

Reply:
I think not all Linux/Posix systems have it.